app/Http/Controllers/Api/V1/RoundController.php

```diff
@@ -33,6 +33,43 @@ public function show(Round $round): RoundResource
     {
         return new RoundResource($round);
     }
+
+    public function store(Request $request)
+    {
+        // Convert 'when' to the correct format
+        $requestData = $request->all();
+        $requestData['when'] = (new \DateTime($requestData['when']))->format('Y-m-d H:i:s');
+    
+        // Set the host_id to the authenticated user's ID
+        $requestData['host_id'] = Auth::id();
+    
+        // Create the round
+        $round = Round::create($requestData);
+    
+        // Automatically add the host as a golfer to the round
+        $userId = Auth::id();
+        $round->users()->attach($userId, ['status' => 'accepted']); // Use 'attach' to add the golfer
+    
+        return new RoundResource($round);
+    }
+   
+    public function update(Request $request, Round $round)
+    {    
+        // Convert 'when' to the correct format
```

Comment:
same... need validation.  if its the same validation, use the same request that you use for store